snapshot_dbg_cli/delete_debuggees_command.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the support for the delete_debuggees command.
+
+The delete_debuggees command is used to delete debuggees. Deleting a debuggee
+will also delete all breakpoints that belong to the debuggee.
+"""
+
+from snapshot_dbg_cli.exceptions import SilentlyExitError
+from snapshot_dbg_cli.debuggee_utils import sort_debuggees
+from snapshot_dbg_cli.time_utils import get_current_time_unix_msec
+
+DESCRIPTION = """
+Used to delete debuggees. Deleting a debuggee will also delete all breakpoints
+that belong to the debuggee.  You are prompted for confirmation before any
+debuggees are deleted. To suppress confirmation, use the --quiet option. By
+default only stale debuggees will be deleted. To include other debuggees for
+deletion include either the --include-inactive or --include-all flags.  A
+debuggee is considered stale if it has not run for the past 7 days.
+"""
+
+ID_HELP = """
+Zero or more debuggee IDs. The specified debugges will be deleted. By default,
+if no debuggees IDs are specified, all stale debuggees are selected for
+deletion.  A debuggee is considered stale if it has not run for the past 7 days.
+"""
+
+INCLUDE_INACTIVE_HELP = """
+If set, include all inactive debuggees. A debuggee is considered to be inactive
+if it has not run in the past 5-6 hours. By default, only stale debuggees will
+be deleted. This flag is not required when specifying the exact ID of a
+debuggee.
+"""
+
+INCLUDE_ALL_HELP = """
+If set, include all debuggees. By default, only stale debuggees will be deleted.
+This flag is not required when specifying the exact ID of a debuggee.
+"""
+
+QUIET_HELP = 'If set, suppresses user confirmation of the command.'
+
+SUMMARY_HEADERS = ['Name', 'ID', 'Last Active', 'Status']
+
+
+def transform_to_debuggee_summary(debuggee):
+  # Match the fields from SUMMARY_HEADERS
+  return [
+      debuggee['displayName'],
+      debuggee['id'],
+      debuggee['lastUpdateTime'],
+      get_debuggee_status(debuggee),
+  ]
+
+
+def get_debuggee_status(debuggee):
+  if not debuggee['activeDebuggeeEnabled']:
+    return 'WARNING UNKNOWN'
```

Comment:
I'd prefer just "UNKNOWN"

Reply:
Fair, this ties into your comment below about a more explicit warning.